File: 6 Pattern Printing/2_pattern_1.cpp
// print row and column
// row
// 1 1 1 1
// 2 2 2 2
// 3 3 3 3
// 4 4 4 4

#include<iostream>
using namespace std;

int main()
{
    int row, col;
    
    for(row=1; row<6; row++)
    {
        for(col=1; col<6; col++)
        {
            cout<<row<<" ";
        }
        cout<<endl;
    }
}



// column
// 1 2 3 4
// 1 2 3 4
// 1 2 3 4
// 1 2 3 4

// #include<iostream>
// using namespace std;

// int main()
// {
//     int row, col;
    
//     for(row=1; row<6; row++)
//     {
//         for(col=1; col<6; col++)
//         {
//             cout<<col<<" ";
//         }
//         cout<<endl;
//     }
// }

File: 006_Pattern_Printing/2_pattern_1.py

# print row and column
# row
# 1 1 1 1
# 2 2 2 2
# 3 3 3 3
# 4 4 4 4

n = 5
for i in range(1, n):
    for j in range(1, n):
        print(i, end=" ")
    print()

print()


# column
# 1 2 3 4
# 1 2 3 4
# 1 2 3 4
# 1 2 3 4

n = 5
for i in range(1, n):
    for j in range(1, n):
        print(j, end=" ")
    print()